packages/components/src/guide/types.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import type { ModalProps } from '../modal/types';
+
+export type Page = {
+	/**
+	 * Content of the page.
+	 */
+	content: ReactNode;
+	/**
+	 * Image displayed above the page content.
+	 */
+	image?: ReactNode;
+};
+
+export type GuideProps = {
+	/**
+	 * @deprecated since 5.5 – use `pages` prop instead
+	 */
+	children?: ReactNode;
```

Comment:
It's a bit redundant in some contexts, but could we add the same text outside of the `@deprecated` tag so it also shows up in the Storybook props table?

```suggestion
	/**
	 * Deprecated. Use `pages` prop instead.
	 *
	 * @deprecated since 5.5
	 */
	children?: ReactNode;
```